commands/util/MigrationCommand.ts

```diff
@@ -57,32 +57,30 @@ export default abstract class MongodbMakeMigration extends BaseCommand {
             }
           }),
       )
-    )
-      .filter((migrationFile) => migrationFile !== null)
-      .flat() as string[];
+    ).filter((migrationFile) => migrationFile !== null) as unknown) as string[])
```

Comment:
The parenthesis nesting  makes the code really difficult to read

Maybe use an intermediate variable?